docs/kane-ai-automation-code-generation.md

```diff
@@ -0,0 +1,86 @@
+---
+id: kane-ai-automation-code-generation
+title: KaneAI Automation Code Generation
+hide_title: false
+sidebar_label: Automation Code Generation
+description: Explore KaneAI’s features and components to generate automation test code.
+keywords:
+  - lambdatest automation
+  - lambdatest kaneai
+  - kaneai code gen
+url: https://www.lambdatest.com/support/docs/kane-ai-automation-code-generation
+site_name: LambdaTest
+slug: kane-ai-automation-code-generation/
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<script type="application/ld+json"
+      dangerouslySetInnerHTML={{ __html: JSON.stringify({
+       "@context": "https://schema.org",
+        "@type": "BreadcrumbList",
+        "itemListElement": [{
+          "@type": "ListItem",
+          "position": 1,
+          "name": "Home",
+          "item": "https://www.lambdatest.com"
+        },{
+          "@type": "ListItem",
+          "position": 2,
+          "name": "Support",
+          "item": "https://www.lambdatest.com/support/docs/"
+        },{
+          "@type": "ListItem",
+          "position": 3,
+          "name": "KaneAI Code Gen",
+          "item": "https://www.lambdatest.com/support/docs/kane-ai-automation-code-generation/"
+        }]
+      })
+    }}
+></script>
+This is Kane AI's Generate code workplace where you can generate test automation code for App & Web applications in multiple available languages.
```

Comment:
Replace with: KaneAI generates automation script is multiple frameworks and languages based on natural language inputs from users. KaneAI is available for desktop web applications as well as native mobile applications currently.